tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/foo/tests/main_test.bal

```diff
@@ -1,65 +0,0 @@
-// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Any reason to remove this file completely? This file is used in `MultiModuleTestDebugTest` and it is currently disabled now.